napari/_qt/_qapp_model/qactions/_view.py

```diff
@@ -0,0 +1,91 @@
+from typing import List
+
+from app_model.types import Action, KeyCode, KeyMod, StandardKeyBinding
+
+from ...._app_model.constants import CommandId, MenuGroup, MenuId
+from ....settings import get_settings
+from ....utils.translations import trans
+from ...qt_main_window import Window
+from ...qt_viewer import QtViewer
+
+
+def _tooltip_visibility_toggle():
+    settings = get_settings().appearance
+    settings.layer_tooltip_visibility = not settings.layer_tooltip_visibility
+
+
+VIEW_ACTIONS: List[Action] = [
+    Action(
+        id=CommandId.TOGGLE_FULLSCREEN,
+        title=CommandId.TOGGLE_FULLSCREEN.title,
```

Comment:
I do not like this pattern where the object is used twice. First directly, then its attribute.

Reply:
What's your suggestion?  (Since title isn't _always_ going to be linked to the command)

Reply:
If the title is None/empty, then the contractor could check `id` `title` attribute. 
But maybe this may reduce readability?

Reply:
that also assumes that the user of app-model might be using an enum that just happens to have a `title` attribute.  Really, adding `title` was just a nice way to keep all this related stuff together on the napari side... but `app-model` doesn't care whether the _always_ associates a specific command id and title

Reply:
Ok :)